api/envoy/config/route/v3/route_components.proto

```diff
@@ -1164,6 +1164,10 @@ message DirectResponseAction {
   //   :ref:`envoy_api_msg_config.route.v3.Route`, :ref:`envoy_api_msg_config.route.v3.RouteConfiguration` or
   //   :ref:`envoy_api_msg_config.route.v3.VirtualHost`.
   core.v3.DataSource body = 2;
+
+  // Specifies the response flag to record after sending the response.
+  // If the value is set, it must be a valid %RESPONSE_FLAGS% from access log.
```

Comment:
Please add a link to the page where the various response flags are defined/documented. Also, please note that an example use case is setting this to NR.

Reply:
Ack!